samples/unit-tests/data/general/demo.js

```diff
@@ -240,6 +240,20 @@ QUnit.test('Combination charts and column mapping', function (assert) {
         `Name should be mapped correctly from CSV for packed bubble series,
         which is non-cartesian (#19143).`
     );
+
+    chart = Highcharts.chart('container', {
+        data: {
+            csv: 'code;value\n01007;58\n01003;10\n01041;32\n01085;84\n01133;12',
+            columnTypes: ['string', 'float']
```

Comment:
Is `'float'` supported? or should this be `'number'`?

Reply:
`float` is working fine, but I think we should support `number` only.